include/z64scene.h

```diff
@@ -17,6 +19,148 @@ typedef struct {
+        s8 effects; // How the camera reacts during the transition
+    } /* 0x00 */ sides[2]; // 0 = front, 1 = back
+    /* 0x04 */ s16   id;
+    /* 0x06 */ Vec3s pos;
+    /* 0x0C */ s16   rotY;
+    /* 0x0E */ s16   params;
+} TransitionActorEntry; // size = 0x10
+
+typedef struct {
+    /* 0x00 */ u8 spawn;
+    /* 0x01 */ u8 room;
+} EntranceEntry;
+
+typedef struct {
+    /* 0x00 */ u8 ambientColor[3];
+    /* 0x03 */ s8 diffuseDir1[3];
+    /* 0x06 */ u8 diffuseColor1[3];
+    /* 0x09 */ s8 diffuseDir2[3];
+    /* 0x0C */ u8 diffuseColor2[3];
+    /* 0x0F */ u8 fogColor[3];
+    /* 0x12 */ u16 fogNear;
+    /* 0x14 */ u16 fogFar;
+} LightSettings; // size = 0x16
+
+typedef struct {
+    /* 0x00 */ u8 count; // number of points in the path
+    /* 0x04 */ Vec3s* points; // Segment Address to the array of points
+} Path; // size = 0x8
+
+// Mesh headers
+
+typedef enum {
+    /* 0 */ MESH_HEADER_TYPE_0,
+    /* 1 */ MESH_HEADER_TYPE_1,
+    /* 2 */ MESH_HEADER_TYPE_2,
+    /* 3 */ MESH_HEADER_TYPE_MAX
+} MeshHeaderType;
+
+typedef struct {
+    /* 0x00 */ u8 type;
+} MeshHeaderBase; // size = 0x01
+
+typedef struct {
+    /* 0x00 */ Gfx* opa;
+    /* 0x04 */ Gfx* xlu;
+} MeshHeader01Entry; // size = 0x08
+
+typedef struct {
+    /* 0x00 */ MeshHeaderBase base;
+    /* 0x01 */ u8 numEntries;
+    /* 0x04 */ MeshHeader01Entry* entries;
+    /* 0x08 */ MeshHeader01Entry* entriesEnd;
+} MeshHeader0; // size = 0x0C
+
+typedef enum {
+    /* 1 */ MESH_HEADER1_FORMAT_SINGLE = 1,
+    /* 2 */ MESH_HEADER1_FORMAT_MULTI
+} MeshHeader1Format;
+
+typedef struct {
+    /* 0x00 */ MeshHeaderBase base;
+    /* 0x01 */ u8    format; // MeshHeader1Format
+    /* 0x04 */ MeshHeader01Entry* entry;
+} MeshHeader1Base; // size = 0x08
+
+typedef struct {
+    /* 0x00 */ MeshHeader1Base base;
+    /* 0x08 */ void* source;
+    /* 0x0C */ u32   unk_0C;
+    /* 0x10 */ void* tlut;
+    /* 0x14 */ u16   width;
+    /* 0x16 */ u16   height;
+    /* 0x18 */ u8    fmt;
+    /* 0x19 */ u8    siz;
+    /* 0x1A */ u16   mode0;
+    /* 0x1C */ u16   tlutCount;
+} MeshHeader1Single; // size = 0x20
+
+typedef struct {
+    /* 0x00 */ u16   unk_00;
+    /* 0x02 */ u8    id;
+    /* 0x04 */ u32   source;
+    /* 0x08 */ u32   unk_0C;
+    /* 0x0C */ u32   tlut;
+    /* 0x10 */ u16   width;
+    /* 0x12 */ u16   height;
+    /* 0x14 */ u8    fmt;
+    /* 0x15 */ u8    siz;
+    /* 0x16 */ u16   mode0;
+    /* 0x18 */ u16   tlutCount;
+} BgImage; // size = 0x1C
+
+typedef struct {
+    /* 0x00 */ MeshHeader1Base base;
+    /* 0x08 */ u8    count;
+    /* 0x0C */ BgImage* list;
+} MeshHeader1Multi; // size = 0x10
+
+typedef struct {
+    /* 0x00 */ Vec3s pos;
+    /* 0x06 */ s16 unk_06;
+    /* 0x08 */ Gfx* opa;
+    /* 0x0C */ Gfx* xlu;
+} MeshHeader2Entry; // size = 0x10
+
+typedef struct {
+    /* 0x00 */ MeshHeaderBase base;
+    /* 0x01 */ u8 numEntries;
+    /* 0x04 */ MeshHeader2Entry* entries;
+    /* 0x08 */ MeshHeader2Entry* entriesEnd;
```

Comment:
I feel like these should keep the start/end parts somewhere in both names, but it's probably w/e

Reply:
The game uses `numEntries` entries starting at `entries`,
and `entriesEnd` goes unused so I think it can use an inconsistent name